client/DeckRow.jsx

```diff
@@ -3,9 +3,19 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 
 class DeckRow extends React.Component {
+    constructor() {
+        super();
+
+        this.handleDeckClick = this.handleDeckClick.bind(this);
+    }
+
+    handleDeckClick() {
+        this.props.onSelect(this.props.deck);
```

Comment:
Probably sensible to check this exists, especially since the prop isn’t marked as required (even if it was it would only log a warning)